challenge_1/go/rchicoli/reverse.go

```diff
@@ -0,0 +1,10 @@
+package rchicoli
+
+func ReverseString(s string) string {
+	l := len(s) - 1
+	r := make([]byte, l+1)
```

Comment:
Thx for tip @erocs I've improved the script and added an unicode char for testing this. Now everything passes. Looking forward for the next challenges.